src/Error/Error.js

```diff
@@ -0,0 +1,13 @@
+const Error = () => {
```

Comment:
I like that you created this component. There wasn't enough time to implement it so that users see it, but it shows a thoughtful and organized approach to your work that employers will like.